src/main/java/com/microsoft/sqlserver/jdbc/SQLServerMetaData.java

```diff
@@ -60,6 +60,25 @@ public SQLServerMetaData(String columnName, int sqlType, int precision, int scal
         this.scale = scale;
     }
 
+    /**
+     * Constructs a SQLServerMetaData with the column name, SQL type, length.

```

Comment:
I think JavaDoc for this constructor should be more descriptive if we decide to add it to the driver. The description should explain in what cases and why it is necessary to use the new constructor when there is already one for string types. We might also want to update JavaDoc for `public SQLServerMetaData(String columnName, int sqlType)` constructor too, to avoid confusions.

Reply:
there isn't already one for string types, that's why this is added.
I could add the comments into the javadoc. Or do you suggest something else?

Reply:
... ,SQL type, and length.